_data/tools.yml

```diff
@@ -1,3 +1,13 @@
+- name: readmeio/oas
+  category: dsl
+  language: TypeScript / CLI
+  description:
+    A language-agnostic tool for documenting your API right from your code, generating an OpenAPI Definition.
+  link: https://github.com/readmeio/oas
+  v2: true
+  v3: true
+  v3_1: false
```

Comment:
Hi there! I maintain this library and it does in fact support OpenAPI 3.1 (as does [swagger-inline](https://npm.im/swagger-inline)).

Though I should say that the CLI aspect of `oas` is just a wrapper for `swagger-inline` and we recommend using `swagger-inline` directly instead.